package.json

```diff
@@ -4,12 +4,13 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "truffle migrate --reset && truffle test ./test/Simulation.spec.js "
+    "test": "truffle migrate --reset && truffle test ./test/TestTransmuteToken.js "
```

Comment:
The reason migrate is happening here is that the tests are using the deployed contracts instead of the truffle test contracts... this is because of the framework (it takes abis for a given network).

I want to point this out, as normal `truffle test` will fail because the framework won't know about the truffle contract abis. 

we can probably clean this up at some point in the future.